res/controllers/common-hid-packet-parser.js

```diff
@@ -652,26 +652,22 @@ HIDPacket.prototype.send = function() {
     }
 
     for (var group_name in this.groups) {
-        var group = this.groups[group_name];
+        const group = this.groups[group_name]
         for (var field_name in group) {
-            var field = group[field_name];
-            if (field.type=="bitvector") {
```

Comment:
Are you sure it's safe to remove this? I haven't looked to ensure that no existing scripts use it. I think it's safest to make the smallest changes possible to get this working.

Reply:
No script _can_ use it. Please read fully what I have removed. It does nothing except repeatedly assigning `var bit` ^^

Reply:
Oh, I see. Well that's useless.